storage/Devices/LvmVgImpl.cc

```diff
@@ -520,4 +520,102 @@ namespace storage
 	return lhs->get_vg_name() < rhs->get_vg_name();
     }
 
+
+    void
+    LvmVg::Impl::add_dependencies(Actiongraph::Impl& actiongraph) const
+    {
+	// First, all the operations removing or shrinking LVs
+	vector<Actiongraph::Impl::vertex_descriptor> decrease_lv_actions;
+	// Then a sync point
+	Action::Base* decrease_lv_sync = new Action::Create(get_sid(), true);
```

Comment:
No. I tried but Action::Base is an abstract class. Maybe we should make possible to create Base objects or (my preferred solution) have a Action::Sync instead of a boolean attribute.